network-api/networkapi/wagtailpages/tests/buyersguide/test_homepage.py

```diff
@@ -259,3 +259,41 @@ def test_get_editorial_content_index_no_such_page(self):
         result = self.bg.get_editorial_content_index()
 
         self.assertEqual(result, None)
+
+    def test_bg_home_page_with_cta(self):
+        url = self.bg.url
+        response = self.client.get(url)
+        self.assertEqual(response.status_code, 200)
+        self.assertIsNotNone(response.context['featured_cta'])
+
+    def test_bg_home_page_with_no_cta(self):
+        self.bg.call_to_action = None
+        self.bg.save()
+        url = self.bg.url
+        response = self.client.get(url)
+        self.assertEqual(response.status_code, 200)
+        self.assertIsNone(response.context['featured_cta'])
```

Comment:
**Question** Have you considered grouping the lines in the test methods according to the [AAA (arrange, act, assert) pattern](https://automationpanda.com/2020/07/07/arrange-act-assert-a-pattern-for-writing-good-tests/)?

Reply:
After taking a look at the link, updated!